semantic-conventions/syntax.md

```diff
@@ -49,6 +49,7 @@ convtype ::= "span" # Default if not specified
          |   "event"    # see eventspecificfields
          |   "metric"   # (currently non-functional)
          |   "scope"
+         |   "attribute_group"
```

Comment:
Please also add a section below documenting what it's used for and which fields it defines.

Reply:
@arminru I've added the description, can you please take another look? thanks!

Reply:
Thanks!